examples/step-26/doc/results.dox

```diff
@@ -74,20 +74,34 @@ much accuracy.
 The literature has many suggestions on how to choose the time step size
 adaptively. Much can be learned, for example, from the way ODE solvers choose
 their time steps. One can also be inspired by a posteriori error estimators
-that can, ideally, be written in a way that the consist of a temporal and a
+that can, ideally, be written in a way that they consist of a temporal and a
 spatial contribution to the overall error. If the temporal one is too large,
 we should choose a smaller time step. Ideas in this direction can be found,
 for example, in the PhD thesis of a former principal developer of deal.II,
-Ralf Hartmann, published by the University of Heidelberg, Germany, in 2002.
+Ralf Hartmann, published by the University of Heidelberg, Germany, in 2002
+(see @cite Har02).
 
 
 <h4>Better time stepping methods</h4>
 
 We here use one of the simpler time stepping methods, namely the second order
-in time Crank-Nicolson method. However, more accurate methods such as
+in time Crank-Nicolson method. This is surely already better than the even
+more widely used (and even less accurate) implicit Euler method,
+but many other, more accurate methods such as
 Runge-Kutta methods are available and should be used as they do not represent
 much additional effort. It is not difficult to implement this for the current
-program, but a more systematic treatment is also given in step-52.
+program, if one wanted; a more systematic treatment is also given in step-52.
+
+As a general rule, however, one should not be implementing time stepping
+methods by hand as we do here. Rather, one should use one of the available
```

Comment:
I wouldn't write `shouldn't`. There are good reasons to use specialized time-stepping methods for FEM: 1) performance and 2) certain types of problems need specialized time-stepping approaches (dual splitting for incompressible NS), which are often not part of such general libraries. Indeed, for simple cases general-purpose libraries are a very good option.